src/components/v2/appDetails/sourceInfo/environmentStatus/EnvironmentStatus.component.tsx

```diff
@@ -193,7 +193,7 @@ const EnvironmentStatusComponent = ({
                     close={() => {
                         setShowAppStatusDetail(false)
                     }}
-                    appStreamData={appStreamData}
+                    resourcesSyncResult={resourcesSyncResult}
```

Comment:
Why are we passing this and not abstracting it out from appDetails here?

Reply:
done